lgc/include/lgc/state/PalMetadata.h

```diff
@@ -217,6 +218,19 @@ class PalMetadata {
   // The maximum possible value for the spill threshold entry in the PAL meatadata.
   static constexpr uint64_t MAX_SPILL_THRESHOLD = UINT_MAX;
 
+  unsigned int getUserDataCount(unsigned int callingConv);
```

Comment:
nit: can use `unsigned` instead of mixing `unsigned` and `unsigned int`? Also everywhere else.

Reply:
done